core/src/impl/Kokkos_Profiling_Interface.hpp

```diff
@@ -45,19 +45,20 @@
 #define KOKKOSP_INTERFACE_HPP
 
 #include <Kokkos_Macros.hpp>
-
-#if defined(KOKKOS_ENABLE_PROFILING)
-
+#include <cinttypes>
 #include <cstddef>
 #include <Kokkos_Core_fwd.hpp>
 #include <string>
-#include <cinttypes>
 
-#include <impl/Kokkos_Profiling_DeviceInfo.hpp>
 #include <dlfcn.h>
```

Comment:
@dsunder said this needs to be moved back within the KOKKOS_ENABLE_PROFILING guard - some machines will not compile with dynamic libraries